src/popup/components/Truncate.vue

```diff
@@ -50,31 +52,38 @@ export default defineComponent({
     const container = ref<HTMLDivElement>();
     const scroll = ref<HTMLDivElement>();
     const shouldScroll = ref(false);
-    const nameComponent = computed(() => props.str?.endsWith(AENS_DOMAIN) ? props.str.replace(AENS_DOMAIN, '') : '');
+    const nameComponent = computed(() => props.str?.endsWith(AENS_DOMAIN)
+      ? props.str.replace(AENS_DOMAIN, '')
+      : props.str);
 
     const animationTranslate = ref<string>();
     const animationDuration = ref<string>();
 
+    const calculateTruncate = async () => {
+      shouldScroll.value = false;
+      await nextTick();
+      if (!props.fixed && scroll?.value && container.value?.clientWidth) {
+        const { clientWidth } = container?.value;
+        const { clientWidth: scrollWidth } = scroll?.value;
```

Comment:
Why do you check the presence of `container` and `scroll` here despite doing that in the `if` clause?